hudi-common/src/main/java/org/apache/hudi/common/model/CompactionOperation.java

```diff
@@ -75,8 +75,7 @@ public CompactionOperation(Option<HoodieDataFile> dataFile, String partitionPath
       this.dataFileCommitTime = Option.empty();
     }
 
-    this.deltaFilePaths = logFiles.stream().map(s -> s.getPath().toString())
-        .collect(Collectors.toList());
+    this.deltaFilePaths = logFiles.stream().map(s -> s.getPath().getName()).collect(Collectors.toList());
```

Comment:
rename deltaFilePaths to deltaFileNames?

Reply:
Done